### DISCUSSION TITLE
El caracter guion (-) no lo acepta VSC

### DISCUSSION THREAD

**Discussion by JohnZweitens** [atralice/Curso.Prep.Henry#174] [General]
Estimados buen día Feliz Año!! Alguien que me ayude con este tema, en Visual Studio estoy intentando ingresar el comando **git commit -m** para poder confirmar los cambios realizados en el homework pero no me acepta el guion (-). Alguien que me de una mano si le paso también, gracias de antemano.
![ERROR VST (-)](https://user-images.githubusercontent.com/96010417/147877440-fa0fde1a-6866-4d30-961a-e20faec04c01.png)

**Comment by llanesgabrielg** [comment]
git commit -m "COMENTARIO SOBRE LOS CAMBIOS REALIZADOS"

**Reply by JohnZweitens** [reply]
Gracias Amigo por responder!
